request-attrs_test.go

```diff
@@ -49,3 +50,11 @@ func TestRequestAttributes(t *testing.T) {
 	assert.False(t, testFs.FileMode().IsDir())
 	assert.Equal(t, testFs.FileMode().Perm(), os.FileMode(700).Perm())
 }
+
+func TestRequestAttributesEmpty(t *testing.T) {
+	fs, b := getFileStat(math.MaxUint32, nil)
```

Comment:
Is there not a const value which represents all flag values being set? Perhaps we should define one?

Reply:
Fixed!